src/queue/bookmark/index.ts

```diff
@@ -0,0 +1,20 @@
+import { NativeConnection, Worker } from '@temporalio/worker';
+import { createActivities } from './activities';
+import createOrGetConnection from '../../db';
+import { WorkflowQueue } from '../common';
+import { logger } from '../../logger';
+
+export async function run() {
```

Comment:
Let's make a generic executable for temporal that one of its use cases is the bookmark reminder, but we should think about the future here. Similarly we don't create a new index per subscription or notification

Reply:
Yeah, just to note, that would mean we will use a single worker to host all workflows, having this in a separate folder/index, we can register the worker that will solely focus on bookmarks-related tasks. Then register another worker for other scope if we need to. That would allow us to scale per scope (keep bookmarks low, and high amount of workers on heavy computing). I guess I shouldn't be too worried about it for now and handle scalability issues when we get there.

Reply:
Yes, that's actually the dilemma that I was having. Whether to scope this under `bookmark` or `notifications`. I feel `bookmark` scope is a bit small. Maybe we should aim for the bigger one which is `notification`, so every notification workflows are going to be stored here.

Reply:
I didn't mean notification scope, I just meant broader scope for the run function. I don't mind having a bookmarks scope within the temporal (or queue) directory. But the runner needs to be generic. It's not a major issue we can always fix it later

Reply:
Agreed. We can re-organize things when we get there.